doc/source/getting_started/running_mapdl.rst

```diff
@@ -70,6 +70,18 @@ port 50005 with:
 
     /usr/ansys_inc/v211/ansys/bin/ansys211 -port 50005 -grpc
 
+You can configure the IP too. However because of ANSYS limitation to receive
+strings from command line, the IP needs to be read from an external file 
+called ``mylocal.ip``. This file is read automatically.
+
+You can then setup the IP (in linux or Windows-Powershell) with:
+
+.. code::
+    
+    echo "127.0.0.1" > mylocal.ip
+    /usr/ansys_inc/v211/ansys/bin/ansys211 -grpc
```

Comment:
Provide windows example as well.

Reply:
This example works for linux and powershell (windows new terminal), but I didnt check with the traditional cmd.

Reply:
Added Windows CMD and Powershell version in new commit.